### PR TITLE
Bump to latest versions (along with the wasi-sdk-20 builds)

### PR DESCRIPTION
**Pull request by assambar**
- Add latest versions for libxml2, ruby, php, python
 - Made dependencies declarative via `wlr-info.json`, which shall also contain the rest of the information with time (currently spread in wlr-*.sh scripts)
 - Added basic smoke test with sqlite and python.
 - Slim version of ruby and php will be added subsequently and libraries will be published subsequently.

### CODE REVIEW

File: .github/workflows/build-php.yaml [hunk 1]
```diff
@@ -38,6 +38,16 @@ jobs:
             flavor: -slim
             build-php-cli: false
             version: 8.2.0
+          - name: php-
+            suffix: ""
+            flavor: ""
+            build-php-cli: false
+            version: 8.2.6
```
Comment (gzurl) [inline .github/workflows/build-php.yaml:45]: Are we keeping both 8.2.0 and 8.2.6?

**Review by ereslibre — APPROVED** [review]
LGTM, thanks @assambar

File: .github/workflows/build-ruby.yaml [hunk 2]
```diff
@@ -16,6 +16,11 @@ jobs:
         include:
           - version: 3.2.0
             target_version: 3_2_0
+          - version: 3.2.2
+            target_version: 3_2_2
```
Comment (gzurl) [inline .github/workflows/build-ruby.yaml:20]: Same, do we want to support all of them?

**Comment by gzurl** [comment]
@assambar, the PR LGTM, great work!

I would double-check new builds with our current demos.
For instance, we could check PHP 8.2.6 with the Drupal demo by updating the Dockerfile and performing manual inspection (ie: setup, a new page, attaching pictures, etc.).

File: .github/workflows/build-php.yaml (hunk 1 above)
Comment (assambar) [reply .github/workflows/build-php.yaml:45]: For now. Will remove 8.2.6 once we have everything building for 8.2.6 and it is tested with Drupal and WordPress.

**Pull request merged**